odxtools/progcode.py

```diff
@@ -20,6 +20,10 @@ class ProgCode:
     entrypoint: Optional[str]
     library_refs: List[OdxLinkRef]
 
+    @property
+    def code(self) -> bytes:
+        return self._code
```

Comment:
Better to return an object with API similar to `importlib.resources.abc.Traversable`, most PDX files I have seen use `jar` files, for those the bytes are useless, and an actual file is needed.

Reply:
(I hope I didn't misunderstand your point. If so: sorry :/)